server/sonar-web/src/main/js/components/controls/buttons.css

```diff
@@ -66,6 +66,10 @@
   border-color: var(--darkBlue);
 }
 
+.button:disabled,
+.button:disabled:active,
+.button:disabled:focus,
+.button:disabled:hover,
```

Comment:
Is it really necessary to use two pseudo-classes:  
`:disabled:active`
`:disabled:focus`
`:disabled:hover`
??

Reply:
These were present in the 8.5

Reply:
The classes were obtained by viewing the styles present on production